### DISCUSSION TITLE
Just a quick Shoutout

### DISCUSSION THREAD

**Discussion by Ryanman** [rfennell/AzurePipelines#1470] [General]
@rfennell I wanted to take a second and thank you for writing this task. 

It's been a huge enabler for our org, you're clearly involved with fixing issues and answering questions. You've probably saved an enormous amount of time for folks reinventing the wheel, and the tech choices you've made to be crossplat and allow us to define templates just make sense.

Very much appreciated!

**Comment by rfennell** [comment]
@Ryanman thanks for nice words
